src/zck_delta_size.c

```diff
@@ -191,10 +191,13 @@ int main (int argc, char *argv[]) {
         }
         total_size += zck_get_chunk_comp_size(tgt_idx);
     }
-    printf("Would download %lli of %lli bytes\n", (long long) dl_size,
-           (long long) total_size);
-    printf("Matched %lli of %llu chunks\n", (long long) matched_chunks,
-           (long long unsigned) zck_get_chunk_count(zck_tgt));
+    printf("Would download %lli of %lli bytes (%lli%%) in %lli ranges\n", (long long) dl_size,
```

Comment:
Shouldn't this be "in %lli chunks" rather than "in %lli ranges"?  Or am I missing something?

Reply:
good catch. I originally intended to do some kind of chunk to ranges mapping (e.g. counting consecutive chunks to fetch as a single range) but lost time/patience implementing that. changed it to chunks to have consistent naming.